src/main/java/uk/gov/companieshouse/insolvency/data/converter/OffsetDateTimeWriter.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.insolvency.data.converter;
+
+import java.time.OffsetDateTime;
+import java.util.Date;
+
+import org.springframework.core.convert.converter.Converter;
+
+public class OffsetDateTimeWriter implements Converter<OffsetDateTime, Date> {
```

Comment:
Cosmetic: Maybe ***WriteConverter name be more appropriate and follows existing pattern in this package